pjrt/convert/utils/xplane_schema.zig

```diff
@@ -0,0 +1,300 @@
+const std = @import("std");
+const tf_op_utils = @import("tf_op_utils.zig");
+
+pub const HostEventType = enum(u16) {
```

Comment:
please add a comment explaining the casing is important to match the contant of the protobuffer